src/components/avatar/AvatarImage.js

```diff
@@ -1,14 +1,16 @@
 // @flow
-import React from 'react';
+import * as React from 'react';
 
 type Props = {
     className?: string,
+    id?: string | number,
     onError?: Function,
     url: string,
 };
 
-const AvatarImage = ({ className = '', url, onError }: Props) => (
+const AvatarImage = ({ className = '', url, onError, ...rest }: Props) => (
```

Comment:
Would it make sense to destructure `id` explicitly?

Reply:
Woops, `id` should be removed from props, it was meant to be the user id, not the item `id`.